MixItUp.Base/MixerAPI/InteractiveClientWrapper.cs

```diff
@@ -203,6 +203,9 @@ public class InteractiveClientWrapper : MixerWebSocketWrapper
         public Dictionary<string, InteractiveConnectedControlCommand> ControlCommands { get; private set; }
         public LockedDictionary<string, InteractiveParticipantModel> Participants { get; private set; }
```

Comment:
GitHub, once again does NOT show a good diff. This is much easier to see in Visual Studio's diff.